slybot/slybot/plugins/scrapely_annotations/extraction/__init__.py

```diff
@@ -0,0 +1,116 @@
+from scrapely.extraction import (InstanceBasedLearningExtractor,
+                                 _annotation_count)
+from scrapely.extraction.pageparsing import parse_extraction_page
+from scrapely.extraction.pageobjects import TokenDict
+from scrapely.extraction.regionextract import (TraceExtractor,
+                                               TemplatePageExtractor)
+from scrapy.utils.spider import arg_to_iter
+
+from .container_extractors import BaseContainerExtractor, ContainerExtractor
+from .pageparsing import parse_template
+from .region_extractors import BaseExtractor
+from .utils import _count_annotations
+from ..processors import ItemProcessor
```

Comment:
I would leave this `__init__.py` with only the imports of files we want to expose from this `extraction` package in order to do `from .extraction import SlybotIBLExtractor`. I'd move `TemplatePageMultiItemExtractor` and `SlybotIBLExtractor` to their own file. This way, `__init__.py` cleanly describes what can be found in the package and exposes it in a natural way.